ansible/roles/debops.owncloud/defaults/main.yml

```diff
@@ -548,6 +548,9 @@ owncloud__temp_path: ''
 # Where the ownCloud instance will be deployed (web root).
 owncloud__deploy_path: '{{ owncloud__app_home }}'
 
+# .. envvar:: owncloud__deploy_path_mode
+#
+owncloud__deploy_path_mode: '0750'
```

Comment:
Could you add a description of this variable in the comment section as well? it will look better on the documentation page.

Reply:
sure